pkg/svcat/service-catalog/class.go

```diff
@@ -139,7 +139,13 @@ func (sdk *SDK) RetrieveClassByName(name string, opts ScopeOptions) (Class, erro
 	}
 
 	if len(searchResults) == 0 {
-		return nil, fmt.Errorf("class '%s' not found", name)
+		if opts.Scope.Matches(ClusterScope) {
+			return nil, fmt.Errorf("class '%s' not found in cluster scope", name)
+		} else if opts.Scope.Matches(NamespaceScope) {
+			return nil, fmt.Errorf("class '%s' not found in namespace %s", name, opts.Namespace)
```

Comment:
Can you also handle the case where `--all-namespaces` is used?

This is how it behaves currently:
```
 svcat get class foo --all-namespaces --scope namespace
Error: class 'foo' not found in namespace 
```

Reply:
Yes, but the code that you've modified, is also called by `svcat get class`, so your changes shouldn't   "break" it. 

Previously, the get class command would return an appropriate error:
```
$ svcat get class foo --all-namespaces --scope namespace
Error: class 'foo' not found
```

Now, from the users' perspective, the returned error make it seem as though the command is ignoring the `all-namespaces` flag and only looking for the class in the (current) namespace:
```
$ svcat get class foo --all-namespaces --scope namespace
Error: class 'foo' not found in namespace 
```

Reply:
OK, I see. I will change this as you suggested. Thanks!

Reply:
@luksa I have updated the code. Can you please help review again?

Reply:
I believe it should be "not found in any namespace". And I'd personally use a nested if statement, but that's not too big of a deal.

It looks good otherwise.

Reply:
@luksa Thanks! I have updated the code accordingly. Can you please help review again?